crates/routes/src/webfinger.rs

```diff
@@ -44,7 +45,14 @@ async fn get_webfinger_response(
   let community_id: Option<Url> = Community::read_from_name(&mut context.pool(), name, false)
     .await
     .ok()
-    .map(|c| c.actor_id.into());
+    .and_then(|c| {
```

Comment:
Hrm... couldn't you also use `check_community_valid` and do a `result.ok()` here? Not sure also if webfinger should return deleted / removed communities.

Reply:
Done